e2etests/tests/test.js

```diff
@@ -1,11 +1,21 @@
 module.exports = {
-  'test Paysage': function (browser) {
+  'Pushed code is rendered': function (browser) {
     browser.page.programmer()
       .navigate()
       .setCode("top.document.body.appendChild(document.createElement('test-marker'))")
       .click('@go-live')
       .waitForElementPresent('test-marker', 2000);
 
     browser.end();
+  },
+  'Code object is listed': function (browser) {
+    browser.page.programmer()
+      .navigate()
+      .setCodeId('toto')
+      .click('@go-live')
+      .waitForElementPresent('#objects li a:first-of-type', 2000, true, function(result) {
+        this.assert.containsText('#objects li a:first-of-type','toto');
+      });
```

Comment:
You could also use the expect API as follow :
```
browser.page.programmer().expect
      .element('#objects li a:first-of-type').text.to.equal('toto');
```
This example is using a default timeout.